tests/ignite/contrib/metrics/test_average_precision.py

```diff
@@ -87,18 +87,14 @@ def get_test_cases():
 
         test_cases = [
             (torch.randint(0, 2, size=(10,)).long(), torch.randint(0, 2, size=(10,)).long(), 1),
-            (torch.randint(0, 2, size=(100,)).long(), torch.randint(0, 2, size=(100,)).long(), 1),
             (torch.randint(0, 2, size=(10, 1)).long(), torch.randint(0, 2, size=(10, 1)).long(), 1),
-            (torch.randint(0, 2, size=(100, 1)).long(), torch.randint(0, 2, size=(100, 1)).long(), 1),
             # updated batches
             (torch.randint(0, 2, size=(16,)).long(), torch.randint(0, 2, size=(16,)).long(), 16),
```

Comment:
Let's use size = 50 for batched test cases
```suggestion
            (torch.randint(0, 2, size=(50,)).long(), torch.randint(0, 2, size=(16,)).long(), 16),
```